pkg/config/app/kuma-cp/config.go

```diff
@@ -30,29 +28,19 @@ var _ config.Config = &Defaults{}
 
 type Defaults struct {
 	// Default Mesh configuration in YAML that will be applied on first usage of Kuma CP
-	Mesh string `yaml:"mesh"`
+	SkipDefaultMeshCreation bool `yaml:"skipDefaultMeshCreation"`
```

Comment:
1) How about just `SkipMeshCreation`, default is implicit because we are in `defaults` branch. Sorry, I should have written this on slack.
2) Please also add `envconfig:"kuma_defaults_skip_mesh_creation"` so it can be used with env vars